### PR TITLE
components/Publish.vue: Image size warnings

### PR DESCRIPTION
**Pull request by shazow**
image

### CODE REVIEW

File: components/Publish.vue [hunk 1]
```diff
@@ -118,29 +121,65 @@ export default {
...
+    },
     async publish() {
+      await this.checkImage();
+      if (!this.canPublish) return;
+      const ad = this.ad;
       const signer = await this.provider.getSigner();
       const signerAddr = await signer.getAddress();
-      if (signerAddr.toLowerCase() != this.ad.owner) {
-        this.error = 'Incorrect active wallet. Must publish with: ' + this.ad.owner;
+      if (signerAddr.toLowerCase() != ad.owner) {
```
Comment (mveytsman) [inline components/Publish.vue:168]: Might as well
```suggestion
      if (!$address.equal(signerAddr, ad.owner)) {
```

File: components/Publish.vue [hunk 2]
```diff
@@ -74,7 +74,7 @@ input {
         </label>
         <label>
           <span>Image</span>
-          <input type="text" v-model="ad.image" placeholder="https://...." />
+          <input type="text" @change="checkImage" v-model="ad.image" placeholder="https://...." />
```
Comment (mveytsman) [inline components/Publish.vue:77]: I feel like this should be the `onLoad` of the preview image - somehow I feel like doing it on change here may have some race condition where this triggers before the image is loaded?

**Review by mveytsman — APPROVED** [review]
Looks good based on reading but I have yet to exercise it on netlify

File: components/Publish.vue (hunk 2 above)
Comment (shazow) [reply components/Publish.vue:77]: Yea that would be ideal but the image lives inside a different component and that component is loaded 1621 times and I didn't want to add too much junk to it just to do this.

Ideally we'd refactor things to do better.

File: components/Publish.vue (hunk 1 above)
Comment (shazow) [reply components/Publish.vue:168]: Might as well add a change that breaks things? :P 

It's `this.$address` I think, wanna just do all of them in one pass?

Comment (mveytsman) [reply components/Publish.vue:168]: hahaha yeah sure :)

**Pull request merged**